lib/ansible/modules/windows/win_package.py

```diff
@@ -130,6 +130,12 @@
     type: bool
     default: yes
     version_added: '2.4'
+  log_path:
+    description:
+    - Full path of log file.
+    - This file will be kept regardless of overall success or failure
```

Comment:
I will add an extra point here saying this is only valid for MSI files, use `arguments` for other packages or something like that.

Reply:
ok